shepherd.js/rollup.config.mjs

```diff
@@ -45,7 +46,18 @@ const plugins = [
     banner
   }),
   filesize(),
-  visualizer()
+  visualizer(),
+  copy({
```

Comment:
I don't think we want to manually copy files. Was there something we needed an `mts` for? I think `mjs` was all correctly tied to the declaration files and working already for me.